core/src/main/java/com/vml/es/aem/acm/core/script/ScriptScheduler.java

```diff
@@ -61,10 +65,36 @@ public class ScriptScheduler implements Runnable {
 
     private Config config;
 
+    private long intervalBetweenRuns = -1;
```

Comment:
initializer `= -1` is redundant since object creation is managed by OSGi which also managed activate method invocation